### DISCUSSION TITLE
Fixed LOmob nipple and generic penis

### DISCUSSION THREAD

**Discussion by zetathix** [krypto5863/COM-Modular-Installer#14] [General]
Hello CMI,

I fixed

- LOmob nipple which is not exist, work perfectly with LOmob's chikubi and nipple shapekeys group.
- generic uncensor penis (the default one in CMI's 'mbodyXXX.model'), texture, balls weight and propotion to be better on any yotogi scene like footjob or blowjob, etc. They works perfectly. Ex. on footjob, balls will hanging where it should be nicely.

I will do some fine tuning for a little while, after that, where should I send them to submit?

* some feedback for CMI, I wish installer won't check C: for enough space, and do checking on installer file's drive itself would be better.

**Comment by krypto5863** [comment]
I'll give it a look, you can post it in a google drive or mega.nz and I'll check it out!
